apps/base-docs/docs/pages/identity/smart-wallet/sdk/create-coinbase-wallet-sdk.mdx

```diff
@@ -25,8 +27,45 @@ export const provider = sdk.getProvider();
 const addresses = provider.request({method: 'eth_requestAccounts'});
 ```
 
+## Returns
 
-## Returns 
 An object with related SDK methods
 
+```ts
+type CreateAccount = {
+  type: 'create';
+  keys: {
+    type: 'address' | 'p256' | 'webcrypto-p256' | 'webauthn-p256';
+    key: `0x${string}`;
+  }[];
+};
+
+type SubAccountInfo = {
+  address: `0x${string}`;
+  factory?: `0x${string}`;
+  factoryData?: `0x${string}`;
+}
+
+type AddOwnerParams = {
+    address: `0x${string}`;
+    publicKey?: never;
+    chainId: number;
+} | {
+    address?: never;
+    publicKey: `0x${string}`;
+    chainId: number;
+}
+
+type CoinbaseWalletSDK = {
+  getProvider: () => CoinbaseWalletProvider;
+  // Only available v4.4.0+
+  subaccount: {
```

Comment:
I wonder if our default docs should not have anything for sub account, given the extra over head being unnecessary to a lot of teams 🤔

Reply:
Yeah, I could see an argument to just reference the sub-account reference from the features page and keep the default docs more generic.

Reply:
updated here 4e59e03